dev-tools/scripts/smokeTestRelease.py

```diff
@@ -706,11 +707,7 @@ def is_port_in_use(port):
 def testSolrExample(unpackPath, javaPath, isSrc):
   # test solr using some examples it comes with
   logFile = '%s/solr-example.log' % unpackPath
-  if isSrc:
-    os.chdir(unpackPath+'/solr')
-    subprocess.call(['chmod','+x',unpackPath+'/solr/bin/solr', unpackPath+'/solr/bin/solr.cmd', unpackPath+'/solr/bin/solr.in.cmd'])
```

Comment:
The only thing is that this might fail on windows? I'm pretty sure the tar executable bits are lost when unpackaging on windows, but I could be wrong.

Reply:
Now that we run the test from `solr/packaging/build/dev` I think we're ok due to https://github.com/apache/solr/blob/main/solr/packaging/build.gradle#L140 ?